router/router_test.go

```diff
@@ -0,0 +1,13 @@
+package router
```

Comment:
Testing router is a very interesting thing. We should make sure that the routes we defined don't return a 404. So we're testing them very similarly as how we would test a controller, we just need to provide a mocked controller for the router